pkg/utils/utils.go

```diff
@@ -594,3 +595,17 @@ func generateRandomGuid() net.HardwareAddr {
 
 	return guid
 }
+
+// GetWatchNamespace returns the Namespace the operator should be watching for changes
```

Comment:
`os.Getenv("NAMESPACE")` is used in several other files to get operator namespace, why need a separate function?

Reply:
Removed.